assets/js/blocks/classic-template/order-received.tsx

```diff
@@ -0,0 +1,153 @@
+/**
+ * External dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+const isConversionPossible = () => {
+	return false;
+};
+
+const getDescription = () => {
+	return __(
+		'This block represents the <strong>thankyou.php</strong> classic template. The actual rendered template may appear different from this placeholder.',
+		'woo-gutenberg-products-block'
+	);
+};
+
+const getSkeleton = () => {
+	return (
+		<div className="woocommerce-page">
+			<div className="woocommerce-order">
+				<h1>
+					{ __( 'Order received', 'woo-gutenberg-products-block' ) }
+				</h1>
+				<p className="woocommerce-notice woocommerce-notice--success woocommerce-thankyou-order-received">
+					{ __(
+						'Thank you. Your order has been received.',
+						'woo-gutenberg-products-block'
+					) }
+				</p>
+				<ul className="woocommerce-order-overview woocommerce-thankyou-order-details order_details">
+					<li className="woocommerce-order-overview__order order">
+						{ __( 'Order number', 'woo-gutenberg-products-block' ) }
+						: <strong>123</strong>
+					</li>
+					<li className="woocommerce-order-overview__date date">
+						{ __( 'Date', 'woo-gutenberg-products-block' ) }:{ ' ' }
+						<strong>May 25, 2023</strong>
```

Comment:
Maybe needs localisation but this is only a placeholder.

Reply:
@shaunandrews is this something we can neglect?

Reply:
Not essential, I think the idea that this is a date is fine. We could improve by using the current date, along with localizing.

Reply:
Marking this as resolved then, as the placeholder is going to be replaced.